app/src/main/java/com/codesoom/assignment/controllers/ProductController.java

```diff
@@ -32,8 +40,11 @@ public Product detail(@PathVariable Long id) {
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
     public Product create(
+            @RequestHeader("Authorization") String authorization,
             @RequestBody @Valid ProductData productData
     ) {
+        String accessToken = authorization.substring("Bearer".length());
```

Comment:
```suggestion
        String accessToken = authorization.substring("Bearer ".length());
```

이렇게 띄어쓰기도 포함해야되지 않나요?